frontend/src/views/Gallery/Platform.vue

```diff
@@ -219,6 +226,16 @@ onMounted(async () => {
     resetGallery();
     await fetchRoms();
     setFilters();
+
+    // Check if there are query params to set filters
+    if (route.query.filter && route.query.value) {
+      const filter = route.query.filter as FilterType;
+      const value = route.query.value as string;
+      filterToSetFilter[filter](value);
+      onFilterChange(); // Update the UI
+      router.replace({ query: {} }); // Clear query params
```

Comment:
why do we want to clear parameters once filters are loaded?

Reply:
iirc not clearing them out causes some weird behavior, and this was the fix